cli/tests/testdata/error_027_bundle_with_bare_import.ts.out

```diff
@@ -1 +1 @@
-[WILDCARD]error: Relative import path "foo" not prefixed with / or ./ or ../ from "file:///[WILDCARD]/error_027_bundle_with_bare_import.ts"
+[WILDCARD]error: Relative import path "foo" not prefixed with / or ./ or ../
```

Comment:
This is a pretty serious degradation of error messages.

Reply:
It was due to https://github.com/denoland/deno_graph/pull/75 by @nayeemrmn.

It was redundant in quite a few places, but not everywhere.  I don't know if @nayeemrmn had a plan in the places it was removed.

We could revert https://github.com/denoland/deno_graph/pull/75 and re-publish, or just move forward and open an issue to restore the information in the two scenarios it wasn't redundant.

Reply:
If you think we can re-add the info before 1.17, I am happy to merge as is.

Reply:
That might be addressed in https://github.com/denoland/deno/pull/12960 ?

Reply:
Yes #12960 restores it in precisely these two places 
![image](https://user-images.githubusercontent.com/29990554/146019293-707db943-6982-4cfb-87fd-6032dc1c7620.png)
 I don't know exactly why these weren't caught in #12604 and are now but I guess the code is cleaner.